.github/workflows/ai-revision.yaml

```diff
@@ -27,6 +27,8 @@ jobs:
   ai-revise:
     name: AI Revise
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
```

Comment:
Taking a look at available [permission scopes](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions), I think we also want:

```
pull-requests: write
```

This also seems to be what is suggested in the REAMDE at https://github.com/peter-evans/create-pull-request